extension/wallet/src/storage.rs

```diff
@@ -31,6 +29,7 @@ impl Storage {
         Ok(storage.into())
     }
 
+    /// return an item parsing it using `from_str`
```

Comment:
Nit: Rustdoc guidelines suggest a slightly different format: https://doc.rust-lang.org/rustdoc/how-to-write-documentation.html#documenting-components